src/ontology/uberon-edit.obo

```diff
@@ -214638,13 +214640,14 @@ replaced_by: UBERON:0001675
 
 [Term]
 id: UBERON:3011048
-name: genital system
-def: "Anatomical system that has as its parts the organs concerned with reproduction, especially the external genital organs." [https://www.merriam-webster.com/dictionary/genitalia, XAO:0000142]
+name: obsolete genital system
+def: "OBSOLETE Anatomical system that has as its parts the organs concerned with reproduction, especially the external genital organs." [https://www.merriam-webster.com/dictionary/genitalia, XAO:0000142]
```

Comment:
in Mondo, we prepend the definition with OBSOLETE. text... (we add a period after the word obsolete). I spot checked a few examples in the file and it doesn't look like Uberon has prepended definitions with OBSOLETE. But I think this is fine.